### ISSUE TITLE
can't set alpha / transparency anymore

### ISSUE THREAD

**Issue by himijendrix24** [popcornmix/omxplayer#712]
Hello,

currently setting alpha for omxplayer doesn't  seem to work anymore.
I just tried it with a fresh raspiabn 4.14 and omxplayer f06235c.

`omxplayer --alpha 50 ~/test.mp4`

logfile:
[omxplayer.txt](https://github.com/popcornmix/omxplayer/files/3305523/omxplayer.txt)

I have no idea weather it is omxplayer or something else.
I'm using console only, but showing a black backround image with fbi doesn't improve anything.
Also `apt-cache showpkg omxplayer` shows no older versions i could try.

In the past I have used this quiet often to fade videos and found it very useful.

**Comment by jehutting** [comment]
Indeed, it doesn't work. Only when you use ```--win``` option to show the movie in a non-full-screen sized window, the alpha setting works.
@popcornmix: is this the intended behaviour when setting the ```OMX_DISPLAY_ALPHA_FLAGS_DISCARD_LOWER_LAYERS``` flag?

**Comment by himijendrix24** [comment]
Thank you so much, that works fine.
I just use --win 0,0,1920,1080
However I didn't yet check it with the new raspbian busster.

**Comment by johannesvogel** [comment]
Unfortunately I am having the same issue. It used to work before and it took me a few hours to figure out that it wasn't me that caused this. ;-)

Will this be fixed in a future release?

**Comment by magdesign** [comment]
Is there somewhere a recent build of omxplayer available with the transparency issue fixed?
